examples/model_compress/pruning/transformers/transformer_pruning.py

```diff
@@ -0,0 +1,411 @@
+import argparse
```

Comment:
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT license.

Reply:
fixed